main.go

```diff
@@ -189,10 +189,11 @@ func wrappedMain() int {
 		var TTY packer.TTY
 		if !inPlugin {
 			var err error
-			TTY, err = tty.Open()
+			TTY, err := tty.Open()
 			if err != nil {
 				fmt.Fprintf(os.Stderr, "No tty available: %s\n", err)
 			}
```

Comment:
The defer close should probably be an else:
```suggestion
			} else {
				defer TTY.Close()
			}
```

Reply:
Probably yes, but it will be executed in both cases.